docs/tutorials/gp-classification.ipynb

```diff
@@ -0,0 +1,386 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import tinygp\n",
+    "except ImportError:\n",
+    "    %pip install -q tinygp\n",
+    "\n",
+    "try:\n",
+    "    import numpyro\n",
+    "except ImportError:\n",
+    "    %pip uninstall -y jax jaxlib\n",
+    "    %pip install -q numpyro jax jaxlib\n",
+    "\n",
+    "try:\n",
+    "    import arviz\n",
+    "except ImportError:\n",
+    "    %pip install arviz"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(gp-classification)="
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## GP for Classification"
```

Comment:
This should be `# GP for Classification` to get the build to work.

Reply:
And maybe we want to think about this title a bit... maybe just `# Classification` would be better.

Reply:
What is causing the docs build to fail now? 🤔 
https://readthedocs.org/projects/tinygp/builds/17030342/

Reply:
Your branch is quite out of date. You should merge the current main branch into yours to fix the build, I think.